MapboxSearch.xcodeproj/project.pbxproj

```diff
@@ -3711,20 +3707,12 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		042BEB152C2DDFAA0004CD7B /* XCRemoteSwiftPackageReference "mapbox-common-ios" */ = {
-			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/mapbox/mapbox-common-ios.git";
-			requirement = {
-				kind = exactVersion;
-				version = 24.10.0;
```

Comment:
There is no need to specify explicitly the Common SDK version, because the Maps SDK already specifies the exact version for the dependencies